reports/management/commands/import-historical.py

```diff
@@ -8,25 +8,22 @@
 
 class Command(BaseCommand):
     help = 'Import historical GMMP data from an XLSX file'
-    option_list = BaseCommand.option_list + (
-        make_option('--global',
-            action='store_true',
-            help='Coverage is global'),
-        make_option('--region REGION',
-            action='store',
-            dest='region',
-            help='Import historical data for a region. One of: %s' % ', '.join(sorted(REGION_COUNTRY_MAP.keys()))),
-        make_option('--country COUNTRY',
-            action='store',
-            dest='country',
-            help='Import historical data for a country. One of: %s' % ', '.join(sorted(c[0] for c in get_countries()))),
-    )
+    def add_arguments(self, parser):
+        parser.add_argument('--filename', action='append', dest='filename', help='XLSX file to import')
```

Comment:
This seems to be a new option... Any info as to why?

Reply:
Parsing the args kept throwing `unrecognised arguments` errors. Solution was to create `--filename` to hold the file path we're working with. @kilemensi

Reply:
What does that mean @ascii-dev? A quick look at the original code, this is what I understand how the command worked:
```sh
import-historical [--global] [--region <region>] [--country <country>] arg1 arg2 arg3
```

You had to specify one of **global**, **region** or **country** options i.e. **_optional arguments_** and any extra arguments you pass to the command will be treated as filenames and hence the `fname in args` part i.e. **_positional arguments_**

Your current code is making filename also optional arguments and hence me asking why? It should be positional like `poll_ids` in the [docs](https://docs.djangoproject.com/en/2.2/howto/custom-management-commands/)

Reply:
I see what you mean @kilemensi. I'll look into how I can get it sorted. Maybe revert the code and make it work or make `--filename` required.

Reply:
It shouldn't require a revert or any major change @ascii-dev ... it's basically a one line change (or very close to it
```suggestion
        parser.add_argument('filenames', nargs='+', help='Excel file(s) to import')
```

And also change where we're getting `filenames`
```python
        filenames = options['filenames']
```

and you're done

Reply:
Updated the PR to include your suggestions and it works. One thing I noticed though, I started having this new error (I was able to fix it by converting the sheet from `xls` to `xlsx`. 
![image](https://user-images.githubusercontent.com/8318974/100750654-a180ce80-33e6-11eb-9196-c5b76625c5e8.png)

Reply:
@kilemensi ☝🏾

Reply:
Yeah, that's file format issue @ascii-dev